modules/dhcp_common/record/reservation.rb

```diff
@@ -7,6 +7,7 @@ class Reservation < Record
     def initialize(name, ip_address, mac_address, subnet, options = {})
       @type = "reservation"
       @name = name
+      options[:deleteable] = true unless options.key?(:deleteable)
       super(ip_address, mac_address, subnet, options)
```

Comment:
You could avoid modifying the options hash by doing something like:
```
super(ip_address, mac_address, subnet, {:deleteable => true}.merge(options))
```

Reply:
Not sure if you missed this -- could you change how "deleteable" option is initialized?